tools/deps/WorkspacePatcher.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.workspace_patcher;
+
+import com.google.startupos.common.flags.Flags;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.io.IOException;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+// TODO: refactor if more advanced functions/extension is needed
```

Comment:
I think we don't need this comment :)
The code is fine, does what it needs, is simple. We don't need to apologize :)

Please describe what the class does though :)